gapic/templates/testing/constraints-3.7.txt.j2

```diff
@@ -5,7 +5,6 @@
 # Pin the version to the lower bound.
 # e.g., if setup.py has "google-cloud-foo >= 1.14.0, < 2.0.0dev",
 # Then this file should have google-cloud-foo==1.14.0
-google-api-core==1.34.1
```

Comment:
Revert once https://github.com/googleapis/python-api-core/pull/638 is merged and released to PyPI

Reply:
Obsolete